index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <h1 id="logo">Aziz Mejri</h1>
+    <button type="button" class="menu-bars">
+      <i class="fas fa-bars"></i>
+    </button>
```

Comment:
I suggest making the button (hamburger button) inside the `<nav>` tag because the button is also part of the  navigation on this page, and that way it will the button makes sense inside the `<nav>` tag